lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -709,6 +755,9 @@ def wait_for_down(self, vm):
     def __attach_disks(self, entity):
         disks_service = self._connection.system_service().disks_service()
 
+        if self.param('template'):
```

Comment:
i am not familiar with this use case. 

do we want to support that ? 

should i  grab all template disks + disks stated in the module?